### PR TITLE
Added grid3 capacity to the explorer grid statistics

### PR DESCRIPTION
**Pull request by dmahmouali**
changes
- Disable devnet choice
- Added grid3 capacity to the explorer grid statistics

image

issue: https://github.com/threefoldtech/nodes-explorer/issues/12

### CODE REVIEW

**Review by DylanVerstraete — CHANGES_REQUESTED** [review]
Besides 1 typo, you approached this solution in the wrong manner. 

When the user requests stats, the network can be (mainnet, testnet, devnet OR "all") 

Look at the implementation of `getItems` to see how the data is actually fetched. I would start to actually implement the solution there. If you need help or have any additional question, feel free to dm me.

**Comment by DylanVerstraete** [comment]
Either you extend `getItems` or your make sure the stats handler for V3 can also handle network `all`

File: frontend/src/App.vue [hunk 1]
```diff
@@ -24,6 +24,14 @@
             ></v-select>
           </div>
           <v-spacer/>
+          <div class="grid3labal">
```
Comment (xmonader) [inline frontend/src/App.vue:27]: label*

File: frontend/src/App.vue [hunk 2]
```diff
@@ -119,4 +127,10 @@ export default {
   top: 30px !important;
   font-size: 30px !important;
 }
+.grid3labal {
```
Comment (xmonader) [inline frontend/src/App.vue:130]: label*

File: server/controllers/stats.js [hunk 3]
```diff
@@ -1,6 +1,39 @@
...
+          });
+      });
+    })
+  )
+    .then((results) => {
+      for (let i = 0; i < results.length; i++) {
+        results[i] = results[i].data;
+      }
+      const c = [].concat(...results);
+      return c;
```
Comment (DylanVerstraete) [inline server/controllers/stats.js:28]: Write as 

`return results.map(r => r.data)`

File: server/controllers/stats.js [hunk 4]
```diff
@@ -1,6 +1,39 @@
...
+    })
+    .catch((err) => {
+      log.warn(err);
+      return [];
```
Comment (DylanVerstraete) [inline server/controllers/stats.js:32]: Return a httpError

File: server/controllers/stats.js [hunk 5]
```diff
@@ -20,7 +53,26 @@ function computeNodeStats (nodes) {
...
+    nodeSpecs.amountregisteredNodes += nodes.length
+    nodeSpecs.onlinenodes += onlineNodes.length
+    nodeSpecs.cru += sumBy(onlineNodes, node => +node.cru)
+    nodeSpecs.mru += sumBy(onlineNodes, node => Math.ceil(+node.mru/(1024*1024*1024)))
+    nodeSpecs.sru += sumBy(onlineNodes, node => Math.ceil(+node.sru/(1024*1024*1024)))
+    nodeSpecs.hru += sumBy(onlineNodes, node => Math.ceil(+node.hru/(1024*1024*1024)))
+
+    return nodeSpecs
+  } catch (error) {
+    log.warn(error)
```
Comment (DylanVerstraete) [inline server/controllers/stats.js:71]: return httpError

File: server/controllers/stats.js (hunk 3 above)
Comment (DylanVerstraete) [inline server/controllers/stats.js:28]: remove ";"

**Review by DylanVerstraete — CHANGES_REQUESTED** [review]

**Pull request merged**